R/GrieneisenTSData.R

```diff
@@ -0,0 +1,59 @@
+#' Retrieve GrieneisenTS data
+#'
+#' Obtain gut microbiome in Papio cynocephalus from Grieneisen et al. (2021).
+#'
+#' @details
+#' The GrieneisenTS dataset contains 16,234 16S rRNA gene
+#' sequencing-based microbiome profiles from 585 baboon samples
+#' collected over 14 years to determine the heritability of the 
+#' gut microbiome on various environmental factors such as 
+#' diet(high/low diversity),age, season(wet/dry).
+#' Each baboon had an average of 28 samples collected over 4.5 years.
+#' The data set can be used to investigate significance 
+#' of longitudinal sampling at large sample sizes.
+#'
+#' Column metadata includes the samples, baboon ID, collection date of 
+#' the samples, sex(F/M), age, social group, social group size, 
+#' rain per month(mm), season(wet/dry), hydro year, month of the year,  
+#' readcount information, plate information, post PCR DNA(ng) information,  
+#' diets, ASV Shannon's H index, Dietary Shannon's H, Amplicon sequence variant (ASV)
+#' richness, the five first principal coordinates(PCs) ofa Bray-Curtis dissimilarity matrix.

```

Comment:
@YagmurSimsekk can you add here similar explanations (itemized table) than in `HintikkaXOData.R`